### PR TITLE
update doc to reflect latest command

### PR DESCRIPTION
**Pull request by jichenjc**

### CODE REVIEW

**Comment by qiujian16** [comment]
please signoff the commit by

```
git commit --amend -s
git push -f
```
Thanks!

File: content/en/concepts/managedclusterset.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ member clusters to the set.
 Running the following command to add a cluster to the set:
 
 ```shell
-$ clusteradm clusterset set example-clusterset --clusters managed1
+$ clusteradm clusterset add example-clusterset --clusters managed1
```
Comment (qiujian16) [inline content/en/concepts/managedclusterset.md:60]: I think the latest command is to use set? which version of clusteradm are you using?

Comment (jichenjc) [reply content/en/concepts/managedclusterset.md:60]:
```
# clusteradm version
client          version :v0.1.1
server release  version :v1.20.2
```

I followed the guide to download the clusteradm from https://open-cluster-management.io/, not sure it's latest

Comment (qiujian16) [reply content/en/concepts/managedclusterset.md:60]: that might be the released lates clusteramd but not the latest code.

FYI @ycyaoxdu we probably should always consider doc the released version to reduce confusion.

**Pull request merged**